ext/storage_sources/azure_store/test/test_azure_connection.cpp

```diff
@@ -29,6 +29,29 @@
 #define CATCH_CONFIG_MAIN
 #include <catch2/catch.hpp>
 
+#include <stdlib.h>
+#include <iostream>
+
 #include "azure_connection.h"
 
-TEST_CASE("testing class azure_connection", "azure_connection") {}
+// Includes necessary for connection
```

Comment:
I would revert this file, since we don't want commented code in production.

Reply:
reverted :)

Reply:
It doesn't seem to be reverted it yet :(

Reply:
srry mb reverted

Reply:
I still don't quite think this is right, we've not added more code in this file but we've got a bunch more includes? We should only include things in the file where they're needed.

Reply:
I had those added temporarily while i was doing some testing, cleaned up to og template :)

Reply:
Maybe there's something I'm missing, but there are still a bunch of new includes in this patch.

Reply:
cleaned up now tyty :)